diff --git a/dist/shaka-player.compiled.d.ts b/dist/shaka-player.compiled.d.ts
index 9c72e09e9727eac9b32dfce92087b6a4481b1ea7..41bab34bdcd6bbb936c4a76c7a7f7a7dfab96395 100644
--- a/dist/shaka-player.compiled.d.ts
+++ b/dist/shaka-player.compiled.d.ts
@@ -5276,3 +5276,5 @@ declare namespace ಠ_ಠ.clutz {
     static isTypeSupported (keySystem : string , contentType : string ) : boolean ;
   }
 }
+
+export = shaka
