diff --git a/dist/shaka-player.compiled.d.ts b/dist/shaka-player.compiled.d.ts
index 6926ff06abc3c85cb2b12fbe48192508151ac0fa..ab67c0e0fe88181042decda90b93fae3e8480187 100644
--- a/dist/shaka-player.compiled.d.ts
+++ b/dist/shaka-player.compiled.d.ts
@@ -5283,3 +5283,5 @@ declare namespace ಠ_ಠ.clutz {
     static isTypeSupported (keySystem : string , contentType : string ) : boolean ;
   }
 }
+
+export = shaka;
